tests/pw/pages/productsPage.ts

```
@@ -11,6 +11,7 @@
 const productsAdmin = selector.admin.products;
 const productsVendor = selector.vendor.product;
 const vendorTools = selector.vendor.vTools;
+const vendorTools = selector.vendor.vTools;
```

Biome

[error] Shouldn't redeclare 'vendorTools'. Consider to delete it or rename it.

'vendorTools' is defined here:


(lint/suspicious/noRedeclare)